lib/data/multipleBackendGateway.js

```diff
@@ -77,9 +147,36 @@ const multipleBackendGateway = {
                     }
                     cb();
                 });
+            } else if (client.clientType === 'aws_s3') {
+                client.headBucket({ Bucket: client.awsBucketName },
+                err => {
+                    if (err) {
+                        multBackendResp[location] = { error: err };
```

Comment:
If I do that for one possible error, then I have to do it for all of them. I think AWS's errors are well documented enough that they'll be able to figure it out